mlir/lib/Dialect/Linalg/Utils/Utils.cpp

```diff
@@ -870,5 +870,80 @@ getReassociationMapForFoldingUnitDims(ArrayRef<OpFoldResult> mixedSizes) {
   return reassociation;
 }
 
+// Returns true if the value is a constant float or integer.
```

Comment:
Constant or "zero"? Also, is this only meant for scalars?

Reply:
Removed this.